### DISCUSSION TITLE
Multiselect inside Modal / Dialog does not go over modal / dialog

### DISCUSSION THREAD

**Discussion by TipluAbdulhamid** [mantinedev/mantine#4061] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31342946/231520972-b6568276-9804-45c5-88d3-fec460a12430.png)
When I use multiselect inside a modal, it opens inside the modal and I have to scroll inside the modal, but obviously what I want is that it goes over the modal borders.

I thought I can solve this, by using the "withinPortal" prop, but how do I position the dropdown element then?

I think this issue also exists, when the multiselect is placed inside a container which has its own scrollbar.

What I want is this behaviour (sorry for ugly censorship): 
![image](https://user-images.githubusercontent.com/31342946/231521985-c5ebb727-4f3e-47a9-a42e-a764ad8bc9b6.png)

**Comment by rtivital** [comment, marked as answer]
Set `<MultiSelect withinPortal />`

**Reply by TipluAbdulhamid** [reply]
When I used withinPortal, the dropdown element was positioned incorrectly, but I checked the code again and it was our own fault. Someone set position to fixed for some reason... 🤦

**Reply by TipluAbdulhamid** [reply]
When I use "withinPortal" the dropdown element renders behind the modal, because the modal is implemented as a "dialog" html tag, which renders above everything else (top layer). Is there a way to solve this?

**Reply by rtivital** [reply]
I do not think so. You can use Mantine Modal component instead to fix this issue.

**Reply by TipluAbdulhamid** [reply]
alright, thanks for your time
